angular.json

```diff
@@ -67,7 +67,42 @@
               ]
             },
             "de": {
-              "localize": ["de"]
+              "localize": [
+                "de"
+              ]
+            },
+            "staging": {
+              "fileReplacements": [
+                {
+                  "replace": "src/environments/environment.ts",
+                  "with": "src/environments/environment.stage.ts"
+                }
+              ],
+              "optimization": true,
+              "outputHashing": "all",
+              "sourceMap": false,
+              "extractCss": true,
+              "namedChunks": false,
+              "extractLicenses": true,
+              "vendorChunk": false,
+              "buildOptimizer": true,
+              "budgets": [
+                {
+                  "type": "initial",
+                  "maximumWarning": "2mb",
+                  "maximumError": "5mb"
+                },
+                {
+                  "type": "anyComponentStyle",
+                  "maximumWarning": "6kb",
+                  "maximumError": "10kb"
+                }
+              ]
+            },
+            "de": {
+              "localize": [
+                "de"
+              ]
```

Comment:
duplicate
```suggestion
```

Reply:
in staging we need the fixed url 'https://staging.wirlernenonline.de/relay/graphql' this is used by the pipeline later on, so staging in this context means pipeline installations

production should use the dynamic approach window.location.origin + '/relay',

Reply:
rest done